ORLib/src/main/java/io/openremote/orlib/service/ORFirebaseMessagingService.kt

```diff
@@ -208,7 +208,7 @@ class ORFirebaseMessagingService : com.google.firebase.messaging.FirebaseMessagi
     private fun createActionIntent(
         notificationId: Long?,
         acknowledgement: String?,
-        ORAlertAction: ORAlertAction?
+        oRAlertAction: ORAlertAction?
```

Comment:
I think it will look a bit nicer when the **R** is also lowercase or just remove the OR prefix from the variable name :wink: : 

```suggestion
        orAlertAction: ORAlertAction?
```

Reply:
This is an opinion issue but I implemented the following convention in OpenRemote frontend and try to follow it when possible:

https://stackoverflow.com/questions/65705297/convert-camel-case-to-sentence-case-in-javascript/65705298#65705298